README.md

```diff
@@ -546,20 +556,82 @@ By default statements are in the *OK* section, but issues that need to
 be addressed should go to the *Problem:* sections (and briefly the
 summary at the top).
 
+# Re-Reviewing an approved package
+
+The MIR team and thereby in extension Ubuntu continuously evolves the MIR
+expectations and guidelines. For example, we add rules based on painful lessons
+learned to avoid the same from happening again. In that regard the demand for
+testing, quality and so on might be best practice now, but have not been
+fulfilled by packages that are in main for a long time.
+
+The proposed approach would be to:
+
+1. We will add (doing so is a TODO) a query that helps us to spot packages in main for the longest time without having a recent update on their MIR bugs.
```

Comment:
Could re-review be avoided by comment bumping an MIR bug?

Re-assessing oldest MIRs first sounds fair, but may not be the most relevant. I don't have a better solution.